package.json

```diff
@@ -10,9 +10,11 @@
     "test": "jest tests/unit",
     "lint:fix": "eslint --fix */**/*.js tests/**/*.js",
     "cy:open": "cypress open --env CYPRESS_RELEASE_VERSION=full",
+    "cy:openmvp": "cypress open --env CYPRESS_RELEASE_VERSION=mvp",
     "cy:run": "cypress run --env CYPRESS_RELEASE_VERSION=full",
     "cy:runmvp": "cypress run --env CYPRESS_RELEASE_VERSION=mvp  --spec 'tests/cypress/integration/reportRepair/*Mvp.spec.js'",
     "test:integration": "start-server-and-test dev http://localhost:3000 cy:open",
+    "test:integration:mvp": "start-server-and-test start http://localhost:3000 cy:openmvp",
```

Comment:
nitpick - should we use dev instead of start here?

Reply:
This is a really good question! Tbh I always change it to `dev` when I tun the tests locally. @Bhavesh-Patel what do you think?

Reply:
Yes, makes senses to change it to `dev`.

Reply:
I've replaced it.